source/hardware/controllers/pioneer_ddj_200.rst

```diff
@@ -71,16 +72,22 @@ Mixer section (p. 10)
    :alt: Pioneer DDJ-200 (mixer section)
    :figclass: pretty-figures
 
-   Pioneer DDJ-200 (mixer section)
-
-===  =================================================  ============================================================================================
-No.  Control                                            Function
-===  =================================================  ============================================================================================
-1    :hwlabel:`MASTER CUE` button                       Activates Master output on Headphones
-2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs  Adjust the high/mid/low-frequency regions of the song.
-3    :hwlabel:`CFX` knobs                               Turns on the selected effects
-4    :hwlabel:`HEADPHONE CUE` button                    Toggle PFL for the Channel
-5    Channel faders                                     Adjust the output level for each channel.
-6    Transition FX Button                               Turns on :ref:`AutoDJ <djing-auto-dj>`.
-7    Cross fader                                        Fades between left and right deck.
-===  =================================================  ============================================================================================
+   Mixer section (p. 10)
+
+===  =====================================================  ============================================================================================
+No.  Control                                                Function
+===  =====================================================  ============================================================================================
+1    :hwlabel:`MASTER` button                               Toggle Master/Cue knob of headphones between left and right
+1    :hwlabel:`SHIFT` + :hwlabel:`MASTER` button            Toggle between 2- and 4-deck mode
+2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs      Adjust high/mid/low-frequencies
+3    :hwlabel:`CFX` knobs                                   Turns on the selected effects
+4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
+4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
+4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1`   Load selected track to left deck.
+4    :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 2`   Load selected track to right deck.
+4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1`  Toggle between left deck between 1 / 3 in 4-deck mode (if LED is lit, deck 3 is active).
+4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 2`  Toggle between right deck between 2 / 4 in 4-deck mode (if LED is lit, deck 4 is active).
```

Comment:
Is this correct? Then I think this would be a good clarification.

```suggestion
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 1`  Toggle between left deck between 1 / 3 in 4-deck mode (if LED is lit, deck 3 is active). If 4-deck mode is disabled, this behaves the same as :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 1`.
4    :hwlabel:`SHIFT (right)` + :hwlabel:`HEADPHONE CUE 2`  Toggle between right deck between 2 / 4 in 4-deck mode (if LED is lit, deck 4 is active). If 4-deck mode is disabled, this behaves the same as :hwlabel:`SHIFT (left)` + :hwlabel:`HEADPHONE CUE 2`.
```